From 45bf6f8dabcd523ca29bb3e332a3f18a1fe7a1df Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ville=20Syrj=C3=A4l=C3=A4?= <ville.syrjala@linux.intel.com>
Date: Fri, 4 Oct 2013 14:53:34 +0300
Subject: drm: Make vblank_inmodeset unsigned
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

vblank_inmodeset is a bitmask, with only two bits mind you, but better
make it unsigned anyway.

Signed-off-by: Ville Syrjälä <ville.syrjala@linux.intel.com>
Reviewed-by: Daniel Vetter <daniel.vetter@ffwll.ch>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit 54edf9aec79779f85d49674580b7ccab4d6f3c4a)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 include/drm/drmP.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/drm/drmP.h b/include/drm/drmP.h
index d3b7d2c96d23..9caba80b95b0 100644
--- a/include/drm/drmP.h
+++ b/include/drm/drmP.h
@@ -1167,7 +1167,7 @@ struct drm_device {
 					/* for wraparound handling */
 	bool *vblank_enabled;           /* so we don't call enable more than
 					   once per disable */
-	int *vblank_inmodeset;          /* Display driver is setting mode */
+	unsigned int *vblank_inmodeset; /* Display driver is setting mode */
 	u32 *last_vblank_wait;		/* Last vblank seqno waited per CRTC */
 	struct timer_list vblank_disable_timer;
 
-- 
1.8.5.rc3

